include/ze_info/zenon.hpp

```diff
@@ -92,6 +92,17 @@ class zenon
     ze_group_count_t group_count = {};
     ze_command_queue_desc_t command_queue_descriptor = {};
     ze_command_queue_handle_t command_queue;
+    
+    ze_command_queue_handle_t input_copy_command_queue;
+    ze_command_list_handle_t input_copy_command_list;
+    ze_command_queue_desc_t input_copy_command_queue_descriptor = {};
+    ze_command_list_desc_t input_copy_command_list_descriptor = {};
+
+    ze_command_queue_handle_t output_copy_command_queue;
```

Comment:
input copy and output copy could be done with one cmdlist synchronized on events. Consider this as next step

Reply:
its in my mind, i think we can add it a bit later , i will add appropietate  task in teams